source/draw/gpu/pc/Window.ooc

```diff
@@ -23,7 +23,7 @@ use ooc-base
 import X11/X11Window
 import X11/include/x11
 
-Window: class extends GpuSurface {
+Window: class extends OpenGLSurface {
```

Comment:
Okay, looking forward to it. I'd like to get input working at some point and it doesn't make sense to have it as part of the OpenGL stuff.